{\rtf1\ansi\ansicpg1252\cocoartf1504\cocoasubrtf760
{\fonttbl\f0\fnil\fcharset0 Menlo-Regular;}
{\colortbl;\red255\green255\blue255;\red0\green0\blue0;\red255\green255\blue255;}
{\*\expandedcolortbl;;\csgray\c0;\csgray\c100000;}
\margl1440\margr1440\vieww10800\viewh8400\viewkind0
\pard\tx560\tx1120\tx1680\tx2240\tx2800\tx3360\tx3920\tx4480\tx5040\tx5600\tx6160\tx6720\pardirnatural\partightenfactor0

\f0\fs22 \cf2 \cb3 \CocoaLigature0 Changed the psswrd\
\
File "/Users/shadmanahmed/Desktop/PYTHON/myEnvironments/py2FlaskEnv/lib/python2.7/site-packages/MySQLdb/connections.py", line 193, in __init__\
    super(Connection, self).__init__(*args, **kwargs2)\
sqlalchemy.exc.OperationalError: (_mysql_exceptions.OperationalError) (1045, "Access denied for user 'root'@'localhost' (using password: YES)")\
\
Changed the usrnm\
\
File "/Users/shadmanahmed/Desktop/PYTHON/myEnvironments/py2FlaskEnv/lib/python2.7/site-packages/MySQLdb/connections.py", line 193, in __init__\
    super(Connection, self).__init__(*args, **kwargs2)\
sqlalchemy.exc.OperationalError: (_mysql_exceptions.OperationalError) (1045, "Access denied for user 'shoot'@'localhost' (using password: YES)")\
\
Changed the port\
\
  File "/Users/shadmanahmed/Desktop/PYTHON/myEnvironments/py2FlaskEnv/lib/python2.7/site-packages/MySQLdb/connections.py", line 193, in __init__\
    super(Connection, self).__init__(*args, **kwargs2)\
sqlalchemy.exc.OperationalError: (_mysql_exceptions.OperationalError) (2003, "Can't connect to MySQL server on '127.0.0.1' (61)")\
\
}